### PR TITLE
UPD: Added debug messages to test_openshift_templates

### PR DESCRIPTION
**Pull request by fgiorgetti**

### CODE REVIEW

File: router/smoke/openshift/inventory-amq-interconnect-1-tls-auth.yml [hunk 1]
```diff
@@ -13,8 +13,8 @@ all:
           router_config: /etc/qpid-dispatch/qdrouterd.conf
           implementation: dispatch
           service: qdrouterd
-          pem_file: client.crt
-          key_file: client-key.pem
+          pem_file: "client.crt"
```
Comment (lenosi) [inline router/smoke/openshift/inventory-amq-interconnect-1-tls-auth.yml:16]: Next time I'm for using only one type of (quotation marks) > apostrophes in document/project.

""

File: router/smoke/openshift/inventory-amq-interconnect-1-tls-auth.yml [hunk 2]
```diff
@@ -31,8 +31,8 @@ all:
         ansible_connection: docker
         component: client
         executor: docker
-        pem_file: 'client.crt'
-        key_file: 'client-key.pem'
-        keystore: 'clientkeystore'
+        pem_file: '/client.crt'
```
Comment (lenosi) [inline router/smoke/openshift/inventory-amq-interconnect-1-tls-auth.yml]: and here ''

File: router/smoke/openshift/inventory-amq-interconnect-1-tls-auth.yml (hunk 1 above)
Comment (fgiorgetti) [reply router/smoke/openshift/inventory-amq-interconnect-1-tls-auth.yml:16]: Done.

**Review by fgiorgetti — COMMENTED** [review]
Applied requested changes.

**Review by lenosi — APPROVED** [review]

**Pull request merged**